### DISCUSSION TITLE
Homework due at the 24th of April

### DISCUSSION THREAD

**Discussion by Big-Eric-Blip** [cafawo/FinancialDataAnalytics#20] [Q&A] [answered]
Hello Mr. Woebbeking!
Do I remember correctly that only the first task of the homework 03_montecarly is due next week (24.4.24)? Or do we have to commit the whole thing?
Best, Eric :)

**Comment by cafawo** [comment, marked as answer]
Hi @Big-Eric-Blip , 

You're correct, only the first part.

Best,
FW
